classes/Cart.php

```diff
@@ -1067,7 +1067,7 @@ public static function cacheSomeAttributesLists($ipa_list, $id_lang)
         );
 
         foreach ($result as $row) {
-            self::$_attributesLists[$row['id_product_attribute'] . '-' . $id_lang]['attributes'] .= $row['public_group_name'] . ' : ' . $row['attribute_name'] . $separator . ' ';
+            self::$_attributesLists[$row['id_product_attribute'] . '-' . $id_lang]['attributes'] .= $row['public_group_name'] . ': ' . $row['attribute_name'] . $separator . ' ';
```

Comment:
Shouldn't that `": "` be translated so in french we could have `" : "` and `": "` in english?

Reply:
Yes :+1: